core/designsystem/src/main/java/com/mifos/core/designsystem/component/MifosTextField.kt

```diff
@@ -0,0 +1,68 @@
+package com.mifos.core.designsystem.component
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.mifos.core.designsystem.R
+import com.mifos.core.designsystem.theme.Black
+import com.mifos.core.designsystem.theme.DarkGray
+
+@Composable
+fun MifosClientDetailsText(icon: ImageVector, field: String, value: String) {
```

Comment:
move to ui if it can reused other parts of app otherwise, move it respective module.